docs/DeveloperGuide.md

```diff
@@ -409,14 +409,14 @@ The following sequence diagram shows how changes are propagated to the `UI` thro
   * Pros: Easy to implement.
   * Cons: Hard to maintain. Each `Command` now needs to know that `CommandResult` can pass data other than for result display.
 
-### [Proposed] UI Implementation
+### UI Implementation
 **Home Screen**
 
-{Show a screenshot after completion, box the components and label class name beside it}
+<img src="images/UiMainScreen.png" width="600" />
 
 The main screen consists of components:
 * `ResultDisplay`, showing the classes the user has added; 
-* `ResultDisplay` consists of `ClassListPanel`, each row representing a class, and `StudentListPanel`, each row representing a student;
+* `ResultDisplay` consists of `ModuleClassListPanel`, each row representing a `ModuleClassCard`, and `StudentListPanel`, each row representing a `StudentCard`;
 * `CommandBox`, the Command Line Interface (CLI) for user to key in command and
```

Comment:
Perhaps I'm misunderstanding, but isn't `ResultDisplay` the textbox that displays `CommandResult`'s message?